src/asmcnc/production/z_head_qc_jig/z_head_qc_3.py

```diff
@@ -71,7 +71,7 @@ def start_calibration_timer(self):
             self.m.jog_absolute_single_axis('Z', self.m.z_max_jog_abs_limit, 750)
             self.m.jog_relative('X', 30, 6000)
             self.m.jog_relative('X', -30, 6000)
-            self.update_time(30 * self.one_minute) # 30 minutes
+            self.update_time(1 * self.one_minute) # 30 minutes
```

Comment:
normally I just change the value of one_minute, and then this doesn't need editing either :)